src/main/scala/diplomacy/LazyModule.scala

```diff
@@ -118,11 +118,27 @@ abstract class LazyModule()(implicit val p: Parameters)
     children.foreach( _.childrenIterator(iterfunc) )
   }
 
+  def childrenFinder(filter: LazyModule => Boolean): LazyModule = {
```

Comment:
should this be called "findChild"?

Reply:
i think it would have to be findExactlyOneChild